nengo_viz/components/netgraph.py

```diff
@@ -12,6 +10,8 @@
 from nengo_viz.disposable_js import infomodal
 import nengo_viz.layout
 
+from .action import create_action
```

Comment:
Possibly...  we should stick with the same standard.  I think I'd prefer ```from nengo_viz.components.action import create_action``` here, but I keep forgetting what standard we've been going with.